### DISCUSSION TITLE
Status of community PRs

### DISCUSSION THREAD

**Discussion by ThomasGoulet73** [dotnet/wpf#9212] [General]
Hey @dotnet/wpf-developers / @dotnet/dotnet-wpf,

Are there any news about the status of community PRs ? There hasn't been merged community PRs since a couple of months and I wanted to know when you will start to merge PRs again. IIRC the last news was that the team was busy preparing for Microsoft Build with the new FluentWindows theme.

There are a couple open PRs labeled with "Queue for test pass" that haven't been merged also.

Thanks

**Comment by dipeshmsft** [comment]
Hey @ThomasGoulet73, hopefully we will get started with review and merge of PRs by the end of this month.

**Reply by ThomasGoulet73** [reply]
Thanks @dipeshmsft

**Reply by Symbai** [reply]
Not surprised it looks like to be next month instead (or the one after, or the one after the one after before 🤷‍♂️)

**Reply by ThomasGoulet73** [reply]
@Symbai: They did merge some community PRs at the end of June (Including 4 of my PRs): https://github.com/dotnet/wpf/commits/main/?since=2024-06-01&until=2024-06-30

**Reply by Symbai** [reply]
Congrats, my PR from 2022 is still not even commented by the team 🤷‍♂️

**Reply by lindexi** [reply]
@Symbai My PR https://github.com/dotnet/wpf/pull/945  2019
